odl/__init__.py

```diff
@@ -54,3 +54,4 @@
 from . import solvers
 from . import tomo
 from . import trafos
+from . import util
```

Comment:
I think we should remove it tbh. util should be "internal" things. Public stuff should be in the others.